.github/workflows/git_help.yml

```diff
@@ -0,0 +1,27 @@
+name: Git - Mark Issue with Help Wanted - actions-ecosystem
+
+on:
+  issues:
+    types:
+      - opened
+      - edited
+      - reopened
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-ecosystem/action-regex-match@v2
+        id: regex-match
+        with:
+          text: ${{ github.event.issue.title }}
+          regex: "help|not work"
+          flags: 'gi'
```

Comment:
Hi @8ear,

What is the intended purpose of this flag?

Reply:
It is only to have a better triage.